.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: python
-sudo: false
+sudo: required
```

Comment:
I see, they've moved to a fully containerized environment so sudo is not a problem anymore. Odd that they force `sudo: required` though. :+1: